experiment_buddy/experiment_buddy.py

```diff
@@ -142,6 +210,13 @@ def register_param(name, value, prefix=""):
         for k, v in hyperparams.items():
             register_param(k, v)
 
+        if "sweep_definition" in hyperparams.keys() and _is_running_on_cluster():
+            # we don't really need to this here, as orion will only be called upon closing.
+            with open(hyperparams["sweep_definition"]) as f:
+                sweep_config = yaml.safe_load(f)
+            self.orion_experiment = build_experiment(hyperparams["orion_id"], storage=sweep_config["storage"])
+            self.sweep_objective = sweep_config["objective"]
```

Comment:
can we also extract the wandb objective?

Reply:
what do you mean? 
using the same name as https://github.com/wandb/examples/blob/ae685eec610fcbf44a783aa5892a0025eedfd050/examples/keras/keras-cnn-fashion/sweep-bayes-hyperband.yaml#L3= ?